articles/openshift/oshift-sco.md

```diff
@@ -114,90 +105,172 @@ Containers in OpenShift by default don't persist data. Every time an application
+Kubernetes was not designed with stateful applications like databases in mind. For enterprise applications, these systems have non-negotiable business requirements like high availability, data security, disaster recovery, strict performance SLAs and hybrid/multi-cloud operations.
 
-By default, UKCloud will provide 90 day Let's Encrypt SSL certificates for your OpenShift cluster. At the point of deployment, you can to provide your own certificates for use with OpenShift.
+To address the challenge above, UKCloud offers a [Portworx Enterprise](../openshift/oshift-sd-portworx.md) service, assisting with the adoption of Kubernetes for mission critical enterprise and traditional/legacy applications.
 
-### Container native storage
+### Backup/recovery and disaster recovery
 
-Kubernetes was not designed with stateful applications like databases in mind. For enterprise applications, these systems have non-negotiable business requirements like high availability, data security, disaster recovery, strict performance SLAs and hybrid/multi-cloud operations.
+As standard, localised component failures are tolerated within the infrastructure through elimination of single points of failure (including physical server failure or disk failure).
 
-In order to address the challenge above, UKCloud offers a [Portworx Enterprise](../openshift/oshift-sd-portworx.md) service, assisting with the adoption of Kubernetes for mission critical enterprise and traditional/legacy applications.
+Although OpenShift is designed to deploy and manage stateless applications (applications that can be destroyed and re-instantiated without risk of data loss), you should maintain a master copy or backup copy of any persistent or dynamic data hosted on this service (such as MySQL DB) by using, for example, UKCloud's [Cloud Storage](../cloud-storage/cs-sd.md) service.
 
-## Platform management
+### Service limitations
+
+#### Scalability
+
+As a true cloud platform, UKCloud for Managed OpenShift provides full elasticity and scalability. However, to protect the integrity of the platform and manage customer spend, soft limits on the number and size of application instances will be in place. These limits may be extended on request.
 
-UKCloud for OpenShift is managed directly by the customer, using OpenShift's dashboard, the UKCloud Portal, and APIs associated with both.
+#### Persistent volumes
 
-UKCloud maintains responsibility for routine patching and upgrades of the OpenShift platform in-line with the published [OpenShift life cycle and patch management process](oshift-ref-lifecycle.md).
+In line with current restrictions on the OpenStack service that underpins OpenShift, you can claim/attach 25 additional persistent volume claims (PVCs) to each worker node.
+
+### Published features not currently available
+
+The following features published in the [UKCloud for Managed OpenShift Service Definition](oshift-sd.md) are not currently available:
+
+- Janet, PSN and RLI connectivity
```

Comment:
Did we update the SD and therefore not need this line?

Reply:
Service definition still mentions these networks, so I guess we need to leave this in for now